js/src/admin/components/MailPage.js

```diff
@@ -29,12 +31,15 @@ export default class MailPage extends Page {
         {}
       );
 
-      Object.keys(this.driverFields).flatMap(key => this.driverFields[key]).forEach(
-        key => {
-          this.fields.push(key);
-          this.values[key] = m.prop(settings[key]);
+
+
+    for (let driverName in this.driverFields) {
+        for (let driverOptionKey in this.driverFields[driverName]) {
+            this.fields.push(driverOptionKey);
+            this.values[driverOptionKey] = m.prop(settings[driverOptionKey]);
         }
-      );
+    }
```

Comment:
Does this change any functionality or did you just change the loop?

Reply:
It's just other loop, because we need to iterate over an object.